packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -136,7 +136,11 @@ async function getLatestMendixVersion() {
         const dockerTagsResponse = await fetch(
             "https://registry.hub.docker.com/v1/repositories/mendix/runtime-base/tags"
         );
-        const runtimeVersions = (await dockerTagsResponse.json()).map(r => r.name.split("-")[0]);
+        const runtimeVersions = (await dockerTagsResponse.json())
+            .map(r => r.name.split("-")[0])
+            .filter(runtimeVersion => runtimeVersion.startsWith("8"));
+
+        console.log("versions", runtimeVersions);
```

Comment:
Is this also intended to be included in this PR, since it's not in https://github.com/mendix/widgets-resources/pull/596/

Reply:
Removed this line. This file has been touched to fix an MX 8 branch specific issue.

Reply:
So it's unrelated to the scope of this PR and more a fly-by? Because I couldn't find anything about it in the commits or PR description.

Reply:
See [this commit](https://github.com/mendix/widgets-resources/pull/597/commits/a85b82f111e86416ff1419efe381b3a0941e1249). It's a small fix to pick the right version of Mendix for running test projects. It was always picking the latest Mendix version. With this PR it picks the latest 8 version, so that e2e tests can pass again.